packages/ui/src/design-system/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "styleguide",
+  "private": true,
+  "description": "Gatsby example site demoing living-style-guide",
+  "author": "scott.eckenthal@gmail.com",
```

Comment:
Needs correcting

Reply:
I changed it a bunch of times and kept getting re-written :worried: but I will take another look.